### PR TITLE
Exact - New Component and Updates

### LINKED ISSUE #6015: Issues with Exact Online and Sendinblue [BUG]
I am still having issues with Exact Online and Sendinblue. I want new contacts in Exact Online to be added directly in Sendinblue and in the correct list. I have set up a workflow, but after several tests, I see that no new contacts appear in Sendinblue when I add them in Exact. Can anyone help?

scherm__afbeelding_2023-04-12_om_16 46 01_720

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #6015

### CODE REVIEW

File: components/exact/sources/new-account-created/new-account-created.mjs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "exact-new-account-created",
+  name: "New Account Created",
+  description: "Emit new event each time a new account is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMAccounts)",
```
Comment (dylburger) [inline components/exact/sources/new-account-created/new-account-created.mjs:7]:
```suggestion
  description: "Emit a new event each time a new account is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMAccounts)"
```

Comment (dylburger) [reply components/exact/sources/new-account-created/new-account-created.mjs:7]: Changed 'Emit new event' to 'Emit a new event' to follow Google's style guide (use articles)

File: components/exact/sources/new-contact-created/new-contact-created.mjs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "exact-new-contact-created",
+  name: "New Contact Created",
+  description: "Emit new event each time a new contact is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMContacts)",
```
Comment (dylburger) [inline components/exact/sources/new-contact-created/new-contact-created.mjs:7]:
```suggestion
  description: "Emit a new event each time a new contact is created. [See the docs](https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=CRMContacts)"
```

Comment (dylburger) [reply components/exact/sources/new-contact-created/new-contact-created.mjs:7]: Changed 'Emit new event' to 'Emit a new event' to follow Google's style guide (use articles)

**Comment by michelle0927** [comment]
I tested both sources, New Account Created and New Contact Created, and both emit historical events as well as newly created events.

Screenshot 2023-04-17 at 12 08 55 PM
Screenshot 2023-04-17 at 12 08 36 PM
Screenshot 2023-04-17 at 12 10 33 PM
Screenshot 2023-04-17 at 12 11 02 PM

**Comment by vunguyenhung** [comment]
Thanks @michelle0927, your tests seem good to me. I'll move this to ready for release

**Pull request merged**